authority-portal-frontend/src/app/pages/catalog-page/catalog-page/catalog-page.component.ts

```diff
@@ -115,6 +117,14 @@ export class CatalogPageComponent implements OnInit, OnDestroy {
       });
   }
 
+  private openDataOfferDetailDialogOnceFromUrl() {
+    const params = this.route.firstChild?.snapshot.params;
+    if (params) {
+      console.log('PARAMS ' + params['connectorId'] + ' ' + params['assetId']);
```

Comment:
You can easily get the params from `private route: ActivatedRoute` and then `route.snapshot.pathParams` 

or is there some issue with the nested routes?

Reply:
Doesn't work. The path-params are part of a child-route and thus not available in `route.pathParams`. I wanted to do this this way in the beginning but Angular is being Angular and is not having it. The child-route approach is necessary because we want to have the url without path-params if on the main catalog page - which is not possible if you define the route to have path-params